services/openvpn/discovery/dto/bandwidth_test.go

```diff
@@ -19,10 +19,10 @@ package dto
 
 import (
 	"encoding/json"
-	"errors"
 	"testing"
 
 	"github.com/mysteriumnetwork/node/datasize"
+	assert2 "github.com/mysteriumnetwork/node/testkit/assert"
```

Comment:
could we rename this to kitassert or something like this instead of the number two?

Reply:
I'm not sure about this one. Is this critical?

Reply:
Renamed `assert` package to `assertkit` to avoid clashes and import aliasing.